### PR TITLE
Added dependencies of modules to managed, aligned versions across repo

### PR DESCRIPTION
**Pull request by viliusl**
what I did:
```bash
npm start deps:unmanaged
```
it gave me list of unmanaged dependencies - the ones that are in `dependencies`, `devDependencies` in modules, but not present in `managedDependencies` of root `package.json`. I formatted those deps, chose one version (latest) if there were multiple versions and added to root `package.json`.

then I ran
```bash
npm start deps:sync
```

which updated version for all modules in `dependencies`, `devDependencies` based on the ones defined in `managedDependencies` in root `package.json`.

Also enabled for this would work automatically as a pre-push hook.

@ronami please:)

### CODE REVIEW

**Comment by yanivefraim** [comment]
From what I remember updating JSDOM broke stuff, but give it a try... (:

File: generators/generator-yoshi/package.json [hunk 1]
```diff
@@ -23,19 +23,19 @@
     "extends": "wix/esnext"
   },
   "dependencies": {
-    "chalk": "^1.0.0",
+    "chalk": "^1.1.3, ^1.0.0",
```
Comment (ronami) [inline generators/generator-yoshi/package.json]: is this intended? can this work?

File: examples/auto-generated/autogenerated-client-react/package.json [hunk 2]
```diff
@@ -24,12 +24,12 @@
     "react-addons-test-utils": "^15.3.2",
     "react-test-renderer": "^15.5.4",
     "express-session": "^1.14.2",
-    "jsdom-global": "^2.1.0",
+    "jsdom-global": "^3.0.2",
```
Comment (ronami) [inline examples/auto-generated/autogenerated-client-react/package.json]: jsdom global 3.x.x requires jsdom as peer dependency, can we add it (possibly via octopus)?

**Comment by viliusl** [comment]
@ronami - closing this for now. I think I tried to do too much here. Need to do it in chunks instead

**Pull request closed without merging**